web_src/less/_base.less

```diff
@@ -1300,3 +1299,16 @@ i.icon.centerlock {
 .ui.sub.header {
     text-transform: none;
 }
+
+/* reduce padding around tab items to save some space */
+/* repo header */
+.ui.tabular.menu {
+    min-height: 36px;
+}
+.ui.tabular.menu .item {
+    padding: 0 10px;
+}
```

Comment:
```suggestion
.ui.tabular.menu {
    min-height: 36px;

    .item {
        padding: 0 10px;
    }
}
```

Reply:
I think I prefer these rules in the flat style. I try to avoid nesting where possible because it's hard match these nested rules with rules in the browser devtools which only display them flat too.

Reply:
Thats why there is map file for. Anyway we use such structures everywhere and that's the idea of using less not css directly

Reply:
I tend to agree with @lafriks here. While excessive nesting might be hard to deal with, this is a rather simple case here and perfect usage for it.

Reply:
I can do it in this case but I generally think all forms of selector nesting are bad, they just produce insanely long selector chains and the worst example is Fomantic itself with it's 700kB CSS file that could probably be 100kB if they would use sane selectors and not nest.

Also, I don't recall we have source maps for CSS enabled and even if we had them, it's just not the same being able to transfer single rules between devtools and sources.

Reply:
I do agree that nesting too much is not good but there is no difference between writing:

in css 

```css
.parent {
  yyy;
}
.parent .child {
  xxx;
}
```

and less:

```css
.parent {
  yyy;
  .child
    xxx;
  }
}
```

both will produce same result

Reply:
Nesting done, thought I don't like it.